index.js

```diff
@@ -257,6 +257,26 @@ const execa = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	const originalKill = spawned.kill.bind(spawned);
+	spawned.kill = (signal = 'SIGTERM', options = {}) => {
+		const killResult = originalKill(signal);
+		if ((typeof signal === 'string' && signal.toUpperCase() === 'SIGTERM') ||
+			signal === os.constants.signals.SIGTERM) {
+			if (options.forceKill !== false && killResult) {
```

Comment:
Should we go crazy and combine those two `if` statements into a single one? :)

Also as @sindresorhus suggested, it might be a good idea to extract that into a separate function that returns a boolean.

Reply:
I think we should. :)
Done: 451a743ef2db8784edd87580ce0741ae91bf8e56